caravel/forms.py

```diff
@@ -670,6 +670,22 @@ def __init__(self, viz):
                             "https: //github.com/mbostock/\n"
                             "d3/wiki/Formatting")
             }),
+            'x_axis_formats': (FreeFormSelectField, {
+                "label": _("X axis format"),
+                "default": '.3s',
+                "choices": [
+                    ('.3s', '".3s" | 12.3k'),
```

Comment:
Let's refactor the choices out as a constant `AXIS_FORMAT_CHOICES` instead of copy pasting.